src/main/java/keycontacts/logic/Messages.java

```diff
@@ -53,4 +54,8 @@ public static String format(Student student) {
     public static <T> String format(Collection<T> collection) {
         return collection.stream().map(T::toString).collect(Collectors.joining(", "));
     }
+
+    public static String format(Lesson lesson) {
+        return lesson.toDisplay();
+    }
```

Comment:
There should be consistency in the output messages. Currently, some use Messages.format(lesson), others use lesson.toDisplay(). We should refactor all of them to be the same.